src/css/application.css

```diff
@@ -315,6 +315,20 @@ body {
   border-bottom-left-radius: 0;
 }
 
+.top-bar__project-button-hidden {
+  visibility: hidden;
+}
+
+.top-bar__project-saved {
+  position: absolute;
+  visibility: hidden;
+  left: 1em;
```

Comment:
Why not just do `top`/`left`/`bottom`/`right` all at `0` to pin it to the container dimensions, and then center the text?